src/OrchardCore/OrchardCore.Application.Cms.Core.Targets/OrchardCore.Application.Cms.Core.Targets.csproj

```diff
@@ -1,39 +1,38 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(AspNetCoreTargetFramework)</TargetFramework>
      <!-- NuGet properties-->
     <Title>Target for Modular OrchardCore CMS Application</Title>
     <Description>$(OCCMSDescription)
-    
+
     Converts the application into a modular OrchardCore CMS application with TheAdmin theme but without any front-end Themes.</Description>
     <PackageTags>$(PackageTags) OrchardCoreCMS CMS</PackageTags>
   </PropertyGroup>
 
   <!-- Add ".props" and ".targets" files in the package specific to the "Cms.Core" bundle -->
   <ItemGroup>
     <None Include="OrchardCore.Application.Cms.Core.Targets.props" Pack="true">
-      <PackagePath>build\$(TargetFramework)\OrchardCore.Application.Cms.Core.Targets.props</PackagePath>
+      <PackagePath>build\OrchardCore.Application.Cms.Core.Targets.props</PackagePath>
```

Comment:
Here and in all other places below, can you try this nuspec convention

buildMultitargeting | (4.0) .props and .targets in the buildMultitargeting folder, for cross-framework targeting
-- | --

     <PackagePath>buildMultitargeting\OrchardCore.Application.Cms.Core.Targets.props</PackagePath>

Reply:
Sure thing.